docs/concepts/llms.mdx

```diff
@@ -25,52 +25,53 @@ By default, CrewAI uses the `gpt-4o-mini` model. It uses environment variables i
 - `OPENAI_API_BASE`
 - `OPENAI_API_KEY`
 
-### 2. String Identifier
+<Tabs>
+    <Tab title="String Identifier">
+    ```python Code
+    agent = Agent(llm="gpt-4o", ...)
+    ```
+    </Tab>
 
-```python Code
-agent = Agent(llm="gpt-4o", ...)
-```
-
-### 3. LLM Instance
-
-List of [more providers](https://docs.litellm.ai/docs/providers).
-
-```python Code
-from crewai import LLM
+    <Tab title="LLM Instance">
+    ```python Code
+    from crewai import LLM
 
-llm = LLM(model="gpt-4", temperature=0.7)
-agent = Agent(llm=llm, ...)
-```
+    llm = LLM(model="gpt-4", temperature=0.7)
+    agent = Agent(llm=llm, ...)
+    ```
+    </Tab>
+</Tabs>
 
-### 4. Custom LLM Objects
+### 3. Custom LLM Objects
```

Comment:
![Screenshot 2024-10-30 at 4 15 55 PM](https://github.com/user-attachments/assets/bd25918e-308c-42dc-b7f0-e1ba6569140c)

Numbering is off now.